x/group/keeper/genesis_test.go

```diff
@@ -0,0 +1,227 @@
+package keeper_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	proto "github.com/gogo/protobuf/types"
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/testutil"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/cosmos/cosmos-sdk/x/group/keeper"
+)
+
+type GenesisTestSuite struct {
+	suite.Suite
+
+	app              *simapp.SimApp
+	ctx              context.Context
+	genesisCtx       sdk.Context
+	keeper           keeper.Keeper
+	cdc              *codec.ProtoCodec
+	addrs            []sdk.AccAddress
+	groupAccountAddr sdk.AccAddress
+}
+
+func TestGenesisTestSuite(t *testing.T) {
+	suite.Run(t, new(GenesisTestSuite))
+}
+
+func (s *GenesisTestSuite) SetupSuite() {
+	checkTx := false
+	app := simapp.Setup(s.T(), checkTx)
+	interfaceRegistry := types.NewInterfaceRegistry()
+	group.RegisterInterfaces(interfaceRegistry)
+
+	s.app = app
+	s.genesisCtx = app.BaseApp.NewContext(checkTx, tmproto.Header{})
+	s.keeper = app.GroupKeeper
+	s.cdc = codec.NewProtoCodec(app.InterfaceRegistry())
+	s.ctx = sdk.WrapSDKContext(s.genesisCtx)
+	s.addrs = simapp.AddTestAddrsIncremental(app, s.genesisCtx, 3, sdk.NewInt(30000000))
+	s.groupAccountAddr = s.addrs[0]
+	s.Require().NoError(testutil.FundAccount(s.app.BankKeeper, s.genesisCtx, s.groupAccountAddr, sdk.Coins{sdk.NewInt64Coin("test", 10000)}))
```

Comment:
why do we need that? `AddTestAddrsIncremental` should also give those accounts an initial balance